src/index.ts

```diff
@@ -35,38 +40,30 @@ import {
   WINDOW_S,
   WINDOW_X,
 } from './constants'
-import EmuContainer from './emulator'
-import Transport from '@ledgerhq/hw-transport'
 
-const Resolve = require('path').resolve
-const rndstr = require('randomstring')
+import EmuContainer from './emulator'
+import GRPCRouter from './grpc'
 
-export const DEFAULT_START_OPTIONS = {
+export const DEFAULT_START_OPTIONS: StartOptions = {
   model: DEFAULT_MODEL,
   sdk: '',
   logging: false,
-  X11: false,
   custom: '',
   startDelay: DEFAULT_START_DELAY,
-  pressDelay: DEFAULT_KEY_DELAY,
   startText: 'Ready',
   caseSensitive: false,
   startTimeout: DEFAULT_START_TIMEOUT,
 }
 
 export class StartOptions {
-  model = 'nanos'
-  sdk = ''
   logging = false
-  /**
-   * @deprecated [ZEMU] X11 support is deprecated and not supported anymore
-   */
-  X11 = false
-  custom = ''
   startDelay = DEFAULT_START_DELAY
+  custom = ''
+  model = DEFAULT_MODEL
+  sdk = ''
   startText = 'Ready'
```

Comment:
can we move Ready to a DEFAULT_START_TEXT constant?

Reply:
we should, yes!